WikiWatershed/model-my-watershed
# Unquote arrays to fix test

## Overview

The quotes were causing some tests to fail in Chrome and Safari, since the value `"[]"` would be parsed as a string not an array, but not in Firefox which automatically casted the string to an empty array.

## Testing Instructions

Run `./scripts/testem.sh` in Chrome and Firefox, ensure that all tests pass.

Connects #705
